iothub/service/samples/PnpServiceSamples/Thermostat/Program.cs

```diff
@@ -12,16 +13,29 @@ public class Program
         private static readonly string s_deviceId = Environment.GetEnvironmentVariable("DEVICE_ID");
 
         // These are values as defined in DTMI used for PnP no Component device client sample.
+        // DTDL interface used: https://github.com/Azure/opendigitaltwins-dtdl/blob/master/DTDL/v2/samples/Thermostat.json
         private const string PropertyName = "targetTemperature";
         private const double PropertyValue = 60;
         private const string MethodToInvoke = "reboot";
```

Comment:
"reboot" doesn't exist for `Thermostat` model

Reply:
yes tbd for thermostat. I am aware of this change.